content/admin/identity-and-access-management/using-saml-for-enterprise-iam/managing-team-synchronization-for-organizations-in-your-enterprise.md

```diff
@@ -59,10 +59,11 @@ You can also configure and manage team synchronization for an individual organiz
 8. To disable team synchronization, under "Team synchronization", click **Disable team synchronization**.
 
 {% ifversion team-sync-manage-org-invites %}
-## Managing whether team synchronization can invite non-members to organizations
+## Managing whether team synchronization can re-invite non-members to organizations
+Disabling this setting will let Team Sync re-send invites to non-members if they are in a synced identity group and were previously an organization member removed by an organization owner. 
```

Comment:
To make this article more evergreen, let's remove this UI text. If the UI changes, it will be one less thing for us to update... and folks can get the same information from reading it in the UI.
```suggestion
```